pyEpiabm/pyEpiabm/initial_infected_sweep.py

```diff
@@ -0,0 +1,54 @@
+import random
+from pyEpiabm.infection_status import InfectionStatus
+from .abstract_sweep import AbstractSweep
+
+
+class InitialInfectedSweep(AbstractSweep):
+    """Class for sweeping through population at start
+    of simulation and setting an initial number of infecetd people.
+    """
+
+    def __call__(self, sim_params: dict):
+        """Method that randomly chooses a number of people
+        in the population, changes their infection status to InfectedMild
+        and sets their time of next status change.
+
+        :param sim_params: Dictionary of simulation parameters
+        :type sim_params: dict
+        """
+        pop_size = self._population.total_people()
+        if pop_size < \
+                sim_params["initial_infected_number"]:
+            raise AssertionError('Initial number of infected people needs to be \
```

Comment:
Should this be a value error? (or just assert instead of the if/raise construction)

Reply:
(Ditto below as well)

Reply:
Changed to value errors